data/gegno/gegno ships.txt

```diff
@@ -0,0 +1,1189 @@
+		"SC-14 Plasma Steering"
+		"Hyperdrive"
+	engine -18.5 155.5
+	engine 18.5 155.5
+	turret 0 -87.5 "Astuit Battery"
+	turret -41.5 84.5 "Astuit Battery"
+	turret 41.5 84.5 "Astuit Battery"
+	leak "leak" 60 50
+	leak "flame" 60 80
+	explode "large explosion" 7
+	explode "medium explosion" 20
+	explode "small explosion" 28
+	explode "tiny explosion" 16
+	"final explode" "final explosion medium"
+	description `Felsics are based on Schists, but offer many upgrades in exchange for less durability and space for engines. The Scin have opted for three turret mounts, as well as expanded capacity for outfits and weapons. All this allows these ships to perform logistical duties as well as doubling as a secondary combat vessel, making up for the lack of a dedicated Scin warship.`
+
+ship "Feldspar"
+	sprite "ship/gegno feldspar"
+	thumbnail "thumbnail/gegno feldspar"
+	attributes
+		category "Heavy Warship"
+		licenses
+			"Scin Architect"
+			"Scin Hoplologist"
+		"cost" 14760000
+		"shields" 0
+		"hull" 18800
+		"required crew" 133
+		"bunks" 192
+		"mass" 1090
+		"drag" 15
+		"heat dissipation" 1
+		"fuel capacity" 600
+		"cargo space" 82
+		"outfit space" 566
+		"weapon capacity" 234
+		"engine capacity" 172
+		weapon
+			"blast radius" 180
+			"shield damage" 1650
+			"hull damage" 1200
+			"hit force" 1000
+	outfits
+		"Acuit Artillery" 2
+		
+		"C27 Campaign Core"
+		"R03 Warforge Battery"
+		
+		"SC-15 Plasma Thrusters"
+		"SC-12 Plasma Steering"
+		"SC-14 Plasma Steering"
+		"Hyperdrive"
+	engine 31 194
+	engine -31 194
+	"reverse engine" -36 -186
+		zoom 0.7
+	"reverse engine" 36 -186
+		zoom 0.7
+	"steering engine" 108 34
+		zoom 0.7
+		angle -58
+		left
+	"steering engine" -108 34
+		zoom 0.7
+		angle 58
+		right
+	turret -74.5 13 "Acuit Artillery"
+	turret 74.5 13 "Acuit Artillery"
+	bay "Fighter" -73 -173.5
+		over
+	bay "Fighter" 73 -173.5
+		over
+	bay "Fighter" -70 -98
+		over
+	bay "Fighter" 70 -98
+		over
+	bay "Fighter" -65.5 120
+		angle -180
+		over
+	bay "Fighter" 65.5 120
+		angle -180
+		over
+	leak "leak" 50 50
+	leak "flame" 80 140
+	explode "huge explosion" 18
+	explode "large explosion" 30
+	explode "medium explosion" 46
+	explode "small explosion" 50
+	explode "tiny explosion" 40
+	"final explode" "final explosion large"
+	description `The mighty Feldspar is a Scin creation to behold. As a testbed for the new Halite Fighters, Feldspars bring their own small fleets with them and overwhelm their enemy. They also were designed as a semi-experimental platform, built around a brand new multi-engine system and the latest in Gegno plasma technology.`
+	description `	Able to keep a decent stance away from a battle while putting down artillery fire, Feldspars are quite a dangerously well-rounded ship for their size. On the downside, they are more susceptible to hull damage like any other Scin warship, especially when alone. As such, they rely on their fighters since they aren't a dedicated warship themselves.`
+
+ship "Muscovite"
```

Comment:
While this fits the theme of naming Gegno ships after rocks/minerals, the name for muscovite is based on a well known Earthly location, so it seems weird to use it as a name for an alien ship.